.nvmrc

```diff
@@ -1 +1 @@
-v12.10.0
+v14.16.0
```

Comment:
maybe delete the file and say in the readme to check .tool-versions? then we wouldn't have to keep the files synched

Reply:
I'm okay with leaving it in case someone is using nvm. We have it in FedPack still.